README.md

```diff
@@ -2,45 +2,162 @@
+
+## Packages available
+
+Each service might have a number of libraries available. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/README.md).
+
+The client libraries can be identified by the naming used for their folder, package, and namespace. Each will start with `azure`, followed by the service category, and then the name of the service. For example `azure-storage-blobs`.
 
 For a complete list of available packages, please see the [latest available packages](https://azure.github.io/azure-sdk/releases/latest/#c) page.
 
 > NOTE: If you need to ensure your code is ready for production we strongly recommend using one of the stable, non-beta libraries.
 
-### Client: Previous Versions
-
-Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the beta packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. They might not implement the [guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html) or have the same feature set. They do however offer wider coverage of services.
+### Vcpkg
 
-### VcPkg
+The following SDK library releases are available on [Vcpkg](https://github.com/microsoft/vcpkg):
 
-Released SDK library versions are available on [VcPkg](https://github.com/microsoft/vcpkg):
 * `azure-core-cpp`
 * `azure-identity-cpp`
 * `azure-storage-blobs-cpp`
 * `azure-storage-common-cpp`
 * `azure-storage-files-datalake-cpp`
 * `azure-storage-files-shares-cpp`
 
-> NOTE: In case of getting linker errors when consuming the SDK on Windows, make sure that [VcPkg trilet](https://vcpkg.readthedocs.io/en/latest/users/triplets/) being consumed matches the [CRT link flags](https://docs.microsoft.com/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160) being set for your app or library build. See also `MSVC_USE_STATIC_CRT` build flag.
-
+> NOTE: In case of getting linker errors when consuming the SDK on Windows, make sure that [Vcpkg triplet](https://vcpkg.readthedocs.io/en/latest/users/triplets/) being consumed matches the [CRT link flags](https://docs.microsoft.com/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160) being set for your app or library build. See also `MSVC_USE_STATIC_CRT` build flag.

```

Comment:
```suggestion
> NOTE: In case of getting linker errors when consuming the SDK on Windows, make sure that [VcPkg triplet](https://vcpkg.readthedocs.io/en/latest/users/triplets/) being consumed matches the [CRT link flags](https://docs.microsoft.com/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160) being set for your app or library build. See also `MSVC_USE_STATIC_CRT` build flag.

```

Reply:
This was an intentional casing change to be consistent across the board, and with what the actual `Vcpkg` repo does. I didn't find this mixed casing used anywhere. For example, see https://github.com/microsoft/vcpkg#vcpkg-with-visual-studio-cmake-projects

Reply:
Tha's good, then please make sure we don't have occurrences of using "VcPkg" - I think we have paragraph title called that.

Reply:
Can you be specific if there are any that I missed in this README? I only saw one and fixed it already.